synapse/storage/data_stores/main/__init__.py

```diff
@@ -509,7 +509,7 @@ def get_users_paginate(
         """
         name_filter = {}
         if name:
-            name_filter["name"] = name
+            name_filter["name"] = "%%" + name + "%%"
```

Comment:
why `%%` rather than just `%` ?

Reply:
copy & paste from `_simple_search_list_txn`

Reply:
like almost everything else in #1784, _simple_search_list_txn is doing it wrong.

a single % is fine here